src/GraphKernel.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Graph kernel component with primary methods.
+ *
+ * @mathmodel type Graph is modeled by a set of vertices and a set of edges
+ * @initially <pre>
+ * ():
+ *   ensures
+ *     this is an empty graph with no vertices and no edges.
+ * </pre>
+ */
+public interface GraphKernel {
```

Comment:
This needs to extend Standard. If you are against using the OSU libraries, you can create your own Standard interface with the same three methods.

Reply:
```suggestion
public interface GraphKernel extends Standard<Graph> {
```